src/index.html

```diff
@@ -13,43 +13,91 @@
         </div>
         <h1 class="hotel-name">The Overlook</h1>
       </button>
-      <div class="search-customers hide">
-        <input id="customer-field" type="text" placeholder="Search customer by name" onfocus="this.placeholder=''">
-        <button id="search-customers-button" type="button" name="search-customer"></button>
-      </div>
     </header>
-    <div id="user-bar hide">
-      <h2 id="welcome">Welcome, User</h2>
-      <div class="room-search hide">
-        <div class="room-type-inputs">
-          <label for="room-type-dropdown">Select Room</label>
-          <form class="room-type-dropdown" action="index.html" method="post">
-            <select name="rooms" id="rooms">
+    <main>
+      <div class="user-bar ">
+        <h2 id="welcome">Welcome, User</h2>
+        <div class="search-customers hide">
+          <input id="customer-field" type="text" placeholder="Search customer by name" onfocus="this.placeholder=''">
+          <button id="search-customers-button" type="button" name="search-customer"></button>
+        </div>
+        <div class="room-search">
```

Comment:
Overall you have strong semantics! One thing I notice here is that you have a div in a div. Perhaps, you have a reason, may it be for styling purposes or otherwise. I just wanted to challenge you to consider if this wouldn't be better with a section or article. The reason being is because this is an area I'm definitely trying to improve on.

Reply:
Good catch.  It was for styling, but on a second look it also represents a specific section of the page.  I could use something more semantic.  I'm going to update it.  Thanks!